docs/source/api/users.rst

```diff
@@ -31,13 +31,17 @@ Request Structure
 -----------------
 .. table:: Request Query Parameters
 
+.. versionadded:: ATCv4.1 The ``role`` query parameter was added in version 4.1 of :abbr:`ATC (Apache Traffic Control)`.
```

Comment:
> `WARNING: Content block expected for the "table" directive; none found.`

You added it in between the declaration of a table and its definition. I'm actually shocked that the table still renders fine (I guess it's a table caption with no body and then a floating table with no related caption that just happen to be sequential?)

At any rate, the notice doesn't appear. I see this section already has a `.. versionadded::` for the query parameters added in API v1.4. You should try putting this directly above or below that (separated by a blank line, of course).